### PR TITLE
ad blocker detection, additional cookie banners, and crypto integration

### PR DESCRIPTION
**Pull request by bestadamdagoat**
ad blocker detection (with fun game that lets you skip turning off ad blocker)
* added rightad.js to catch people with ad blockers (as some default lists block it from any domain, didn't wanna phone home to an ad server)
* added blocked.gif because i wanted to encapsulate the beauty of the rizzler on this repo instead of (phoning home) to another gif site

3 cookie banners (broken because something else completely ruined everything on the page, i think it gives a funny charm to the site)
crypto wallet spammer (doesn't do any harm, just asks to connect to a wallet if you have one installed)

### CODE REVIEW

File: rightad.js
Comment (Nasko-5) [inline rightad.js:1]: coconut.jpg

File: rightad.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// purposefully empty do NOT delete pls will make the site annoyingh for everyone
```
Comment (Nasko-5) [inline rightad.js:1]: coconut.jpg

Comment (Dishpit) [reply rightad.js:1]: > coconut.jpg

image

File: blocked.gif
Comment (Dishpit) [inline blocked.gif:1]: shrink file size by a lot pls <3

**Review by Dishpit — CHANGES_REQUESTED** [review]
merge conflicts, pls fix <3

**Pull request merged**